avaframe/out3Plot/statsPlots.py

```diff
@@ -200,3 +206,102 @@ def plotHistCDFDiff(dataDiffPlot, ax1, ax2, insert='True', title=['', '']):
         ax1.set_title(title[1])
 
     return ticks
+
+
+def plotProbMap(avaDir, inDir, cfgFull):
+    """ plot probability maps including contour lines
+
+        Parameters
+        ----------
+        avaDir: str
+            path to avalanche directory
+        inDir: str
+            path to datasets that shall be plotted
+        cfgFull: configParser object
+            configuration settings for probAna
+            keys used: name, cmapType, levels, unit
+
+    """
+
+    # configuration settings
+    cfg = cfgFull['PLOT']
+
+    # fetch probabiltiy map datasets in inDir
+    dataFiles = list(inDir.glob('*.asc'))
+    if dataFiles == []:
+        message = 'No probability map dataset found in: %s' % (inDir)
+        log.error(message)
+        raise FileNotFoundError(message)
+        log.info('Probability maps found for generating plots: %d' % len(dataFiles))
+
+    # load levels and define colors
+    levels = fU.splitIniValueToArraySteps(cfg['levels'])
+    multLabel = False
+    if len(levels) > 2:
+        multLabel = True
+        cmapType = pU.cmapGreys1
+        log.info('in mult')
```

Comment:
Either remove or add a more descriptive log

Reply:
this was just a dev info, removed this